src/rpc/rpcevo.cpp

```diff
@@ -202,10 +202,9 @@ void protx_register_fund_help()
             "5. \"votingKeyAddr\"       (string, required) The voting key address. The private key does not have to be known by your wallet.\n"
             "                         It has to match the private key which is later used when voting on proposals.\n"
             "                         If set to \"0\" or an empty string, ownerAddr will be used.\n"
-            "6. \"operatorReward\"      (numeric, required) The fraction in % to share with the operator. If non-zero,\n"
-            "                         \"ipAndPort\" must be zero as well. The value must be between 0.00 and 100.00.\n"
-            "7. \"payoutAddress\"       (string, required) The dash address to use for masternode reward payments\n"
-            "                         Must match \"collateralAddress\"."
```

Comment:
To my knowledge, the payoutAddress still needs to be the collateralAddress until the DMNs are activated

Reply:
Think that limitation was removed in #2445